home/ivp-planning/machines/selecting-and-moving-objects.md

```diff
@@ -10,24 +10,28 @@ How to select and move objects (german)
 
 To select an object in the [2D-](../user-interface/the-2d-panel.md) or [3D-panel](../user-interface/the-3d-panel.md) it is sufficient to **click on it**.
 
-{% hint style="info" %}
-It is also possible to select multiple objects by holding the **Ctrl** key while clicking on them one after another.
-{% endhint %}
 
-In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blueish tint and a frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel), [rotate ](scale-and-rotate-objects.md#rotating-objects-in-the-2d-panel)or [scale ](scale-objects.md#scaling-objects-in-the-2d-panel)objects in 2D.
+In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blue tint and a white frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel), [rotate ](scale-and-rotate-objects.md#rotating-objects-in-the-2d-panel)or [scale ](scale-objects.md#scaling-objects-in-the-2d-panel)objects in 2D.
```

Comment:
Scaling is currently not supported from the 2D view, so we should remove this. Also lets move the space out of the link here.

```suggestion
In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blue tint and a white frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel) and [rotate](scale-and-rotate-objects.md#rotating-objects-in-the-2d-panel) objects in 2D.
```